docs/dev_guide/example_gallery.rst

```diff
@@ -0,0 +1,20 @@
+.. _example_gallery:
+
+
+Example Gallery
+===============
+
+The purpose of the page is to describe the contribution guidelines for the `SunPy Example Gallery <http://docs.sunpy.org/en/stable/generated/gallery/index.html>`_. All potential contributors to the SunPy Example Gallery should read and abide by the following guidelines.
+
+Contribution Guidelines
+-----------------------
+
+-  The title of the example should be short yet descriptive and emphasize the goal of the example. Try to make the title appeal to a broad audience and avoid referencing a specific instrument, catalog, or anything wavelength dependent.
+-  Each example should begin with a paragraph that gives a brief overview of the entire example, including relevant astronomy concepts, and motivates the described functionality.
+-  The examples must be compatible with the versions supported by the last major release of the SunPy core package (i.e. Python >= 3.6).
+-  All the examples should be checked using one of the many PEP8 linters that are available (autopep8, pylint, flake8 as some examples).
+-  Wherever possible, the examples should include linked references with links pointing to the appropriate ADS entry.
+-  The example should include links to relevant documentation pages.
+-  Each example should include at least one image, map, or plot to use as the icon in the Example Gallery.
```

Comment:
I dont think this should be a hard requirement, there are somethings that are worthy of examples but dont involve plotting. I suggest we reword it to "if possible" or something.

Reply:
sounds good @Cadair -- I incorporated all these suggestions!